### DISCUSSION TITLE
[Feature Request] Desktop app or VsCode extention?

### DISCUSSION THREAD

**Discussion by Sensiblemnd** [AykutSarac/jsoncrack.com#126] [Ideas]
Would be very useful to have it as a Desktop app(electron?) or a vscode extension?

**Comment by TahaScripts** [comment]
Wouldn't be that hard to build this into an Electron app right? It'll be power hungry but it'll be nice for a quick solution.

**Comment by Sensiblemnd** [comment]
I may give it a try

**Comment by Mosaab-Emam** [comment]
I forked this project and will try to make a VS Code extension from it, as well as a desktop app using Tauri.
Wish me luck.

**Reply by Sensiblemnd** [reply]
Just read about Tauri

**Reply by ricosuave82** [reply]
Following and excited about the outcome! :)

**Comment by AykutSarac** [comment]
Hello guys! VS Code extension is currently in progress, you may test it out here! https://github.com/AykutSarac/jsoncrack-vscode
Looking forward to see your contributions!
